docs/assistant/security-assistant.asciidoc

```diff
@@ -72,29 +70,60 @@ NOTE: All chat history and custom quick prompts persist in local browser storage
 
 [discrete]
 [[interact-with-assistant]]
-== Interact with the Security Assistant
+== Interact with the AI Assistant
 
-Use these features to adjust and act on your conversations with the Security Assistant:
+Use these features to adjust and act on your conversations with the AI Assistant:
 
-* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the Security Assistant's answers to be.
+* Select a _system prompt_ at the beginning of a conversation to establish how detailed and technical you want the AI Assistant's answers to be.
 +
 [role="screenshot"]
 image::images/system-prompt.gif[The system prompt drop-down menu,90%]
 +
-NOTE: The system prompt is only configurable at the start of a conversation. To reconfigure it, clear the chat and start a new conversation.
+System prompts provide context to the model, informing its response. To create a custom system prompt, open the system prompts dropdown menu and click *+ Add new system prompt...*.
```

Comment:
I couldn't find the *+ Add new system prompt...* label, but I did see *+ Add quick prompt...* in the BC5:
<img width="1231" alt="prompt" src="https://github.com/elastic/security-docs/assets/79124755/bec68afe-e4e2-413b-b090-f4c3b469a49c">

Reply:
```suggestion
System prompts provide context to the model, informing its response. To create a custom system prompt, open the system prompts drop-down menu and click *+ Add new system prompt...*.
```

Reply:
> I couldn't find the _+ Add new system prompt..._ label, but I did see _+ Add quick prompt..._ in the BC5

**Add new system prompt** is _inside_ the system prompt drop-down. The instructions do say to "open the system prompts drop-down" to click this, but maybe it could be clearer? The animated GIF also shows this, but slightly cuts off the **Add new system prompt** option at the bottom.